custom/icds_reports/migrations/sql_templates/database_views/disha_indicators.sql

```diff
@@ -0,0 +1,110 @@
+DROP VIEW IF EXISTS disha_indicators CASCADE;
+CREATE VIEW disha_indicators AS
```

Comment:
It would be a good convention to use a prefix for all database entities created by custom code. That way they are easily identified in the database. What do you think of renaming to `custom_icds_reports_disha_indicators`?

Reply:
Sure, good idea.